internal/bencodeext/tanglepoint.go

```diff
@@ -49,7 +41,7 @@ func (tp TanglePoint) MarshalBencode() ([]byte, error) {
 	}
 
 	if n := len(tp.Previous); n == 0 {
-		m["previous"] = zeroBendyMessage
+		m["previous"] = Null
```

Comment:
@cryptix You mean here? It's supposed to be `[]byte{0x06, 0x02}`?

Reply:
no, those are just the tangles.

I mean the encoding of previous on the first message on a feed.